base/client.jl

```diff
@@ -71,6 +72,19 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            if lowercase(shell[1]) == "cmd"
+                cmdbuf = IOBuffer()
+                print_cmdshell_escaped(cmdbuf, cmd.exec...)
+                cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
+                cmd.verbatimargument = true
+            elseif lowercase(shell[1]) == "powershell"
+                if length(cmd.exec) >= 2 && cmd.exec[2] == "--%"
+                  cmd.exec[2] = "\"--%\""
```

Comment:
My thought was to quote `--%` in order to only use the native powershell escape mechanism and not the `CMD` fallback.

FWIW I noticed that `--%` only seems to work with commands that are external to powershell (e.g. doesn't work for `dir`).

Thanks for the links.